include/couch_mrview.hrl

```diff
@@ -94,7 +94,10 @@
     resp,
     prepend,
     etag,
-    should_close = false
+    should_close = false,
+    buffer = [],
+    bufsize = 0,
+    threshold = 1490
```

Comment:
@chewbranca Explanation was given in the related PR: https://github.com/apache/couchdb-chttpd/pull/38